standard/StandardProject.cmake

```diff
@@ -18,23 +18,33 @@ include(CMakeParseArguments) # Backwards compatibility
 
 # This must be a macro since project defines scope-local variables
 # that we generally rely on being in the root context.
-macro(standard_project project_name)
```

Comment:
When I test this with a local merge into autowiring, I get the following error :
```
CMake Error at CMakeLists.txt:5 (standard_project):
  Unknown CMake command "standard_project".


-- Configuring incomplete, errors occurred!
```

The problem appears to be the removal of the `standard_project` macro definition on this line, and renaming it to `project` (?)